# Diagnóstico dos dados brutos

In [2]:
from configuration import *
import pandas as pd

In [3]:
df = pd.read_csv(DETECTIONS, sep='|')

In [4]:
n_sources = len(df['audio_source_id'].drop_duplicates())
n_tracks = len(df['track_id'].drop_duplicates())

In [5]:
print('{} rádios\n{} músicas'.format(n_sources, n_tracks))

4987 rádios
101596 músicas


Os dados acima indicam que teremos 101596 *amostras* de 4987 *features* (dimensões). Em média, cada música foi tocada aproximadamente 20 vezes em cada rádio.